app/services/discovery_engine/evaluation/evaluation_resource.rb

```diff
@@ -0,0 +1,67 @@
+module DiscoveryEngine
+  module Evaluation
+    class EvaluationResource
+      attr_reader :sample_set_id
+
+      def initialize(sample_set_id)
+        @sample_set_id = sample_set_id
+      end
+
+      def fetch_quality_metrics
+        create
+        fetch_and_output_metrics
+      end
+
+    private
+
+      attr_reader :evaluation
+
+      def create
+        operation = DiscoveryEngine::Clients
+          .evaluation_service
+          .create_evaluation(
+            parent: Rails.application.config.discovery_engine_default_location_name,
+            evaluation: {
+              evaluation_spec: {
+                query_set_spec: {
+                  sample_query_set: sample_set_name,
+                },
+                search_request: {
+                  serving_config: ServingConfig.default.name,
+                },
+              },
+            },
+          )
+        operation.wait_until_done!
+
+        raise operation.error.message.to_s if operation.error?
+
+        @evaluation = operation.results
+
+        Rails.logger.info("Successfully created evaluation #{evaluation.name}")
+      end
+
+      def fetch_and_output_metrics
+        # TODO: implement a new method in the Metrics::Exported module to send quality metrics to Prometheus instead
+        fetch_with_wait.quality_metrics.to_h
```

Comment:
Let's `Rails.logger.info` this for the time being so it outputs in some way (even though that's not the way it eventually will!)